src/latex/LaTeXWriter.jl

```diff
@@ -48,15 +48,15 @@ considered to be deprecated), or to an empty string if `TRAVIS_TAG` is unset.
 
 See [Other Output Formats](@ref) for more information.
 """
-struct LaTeX <: Documenter.Writer
+mutable struct LaTeX <: Documenter.Writer
     platform::String
-    version::String
-    tectonic::Union{Cmd,String,Nothing}
+    const version::String
+    const tectonic::Union{Cmd,String,Nothing}
```

Comment:
This doesn't work on 1.6. But regardless, I don't see much value in making `.platform` mutable anyway -- just call `hastex()` here in the constructor.

Reply:
Do you mean you'd prefer keeping this an immutable `struct` ?

Reply:
This was reverted in favor of a `hastex()` call in the constructor as suggested